docs/source/models/gbd2017/causes/neural_tube_defects/index.rst

```diff
@@ -97,29 +104,143 @@ The Neural Tube Defects cause has 85 sequelae, which can be summarized as follow
 Cause Model Diagram
 -------------------
 
-There are two possible states for this cause, "with-condition" and
-"without-condition":
+There are two possible states for this cause, **with-condition** for people born *with* neural tube defects,  and
+**free-of-condition** for people born *without* neural tube defects:
 
 .. image:: neural_tube_defects_cause_model_diagram.svg
 
 There is no transition between the states; each person is born into one state or
-the other and permanently stays in that state. Thus, incidence and remission
-rates are zero.
-
-GBD 2017 estimates the prevalence of neural tube defects for every year, sex,
-age, and location combination. These prevalence values can be used to initialize
-the model with the correct fraction of people in each of the two model states.
+the other and permanently stays in that state. **Thus, incidence and remission
+rates are zero.**
 
-People born into the simulation should enter the "with-condition" state
-according to the birth prevalence of neural tube defects; if this data is
-unavailable, we can estimate the birth prevalence using the prevalence and
-mortality rates in the early neonatal age group.
+Data Description
+----------------
 
 .. todo::
 
     Look into data sources. In particular, is birth prevalence data available,
-    or do we need to estimate it using prevalence among the early neonatal age
-    group?
+    or do we need to estimate it using prevalence and mortality rates among the
+    early neonatal age group?
+
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
+.. list-table:: State Definitions
+   :widths: 1, 5, 10
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - C
+     - With **C**\ ondition
+     - Born with neural tube defects
+   * - F
+     - **F**\ ree of Condition
+     - Born without neural tube defects
+
+.. list-table:: State Data
+   :widths: 1, 5, 5, 10
+   :header-rows: 1
+
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - C
+     - prevalence
+     - prevalence_c642
+     -
+   * - C
+     - birth prevalence
+     - birth_prevalence_c642
+     -
+   * - C
+     - excess mortality rate
+     - :math:`\frac{\text{deaths_c642}}{\text{population} \,\times\, \text{prevalence_c642}}`
+     - = (cause-specific mortality rate) / prevalence
+   * - C
+     - disability weight
+     - :math:`\displaystyle{\sum_{s\in \text{sequelae_c642}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+     - = average disability weight over all sequelae
+   * - F
+     - proportion of population
+     - 1 -- prevalence_c642
```

Comment:
For calculations, I am using format prevalence_c_{gbdid}. I will change that if this format (prevalence_c642) is better

Reply:
I vote for fewer underscores, as long as it's still readable. Others can chime in if they have different opinions.